Sources/Testing/ABI/Encoded/ABI.EncodedKnownIssueContext.swift

```diff
@@ -0,0 +1,37 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension ABI {
+  /// A type implementing the JSON encoding of
+  /// ``Issue/KnownIssueContext-swift.struct`` for the ABI entry point and
+  /// event stream output.
+  ///
+  /// This type is not part of the public interface of the testing library. It
+  /// assists in converting values to JSON; clients that consume this JSON are
+  /// expected to write their own decoders.
+  ///
+  /// - Warning: Known issues are not yet part of the JSON schema.
+  struct EncodedKnownIssueContext<V>: Sendable where V: ABI.Version {
+    /// The comment that was passed to `withKnownIssue()`, if any.
+    var comment: Comment?
+
+    /// The source location that was passed to `withKnownIssue()`.
+    var sourceLocation: SourceLocation
+
+    init(encoding context: Issue.KnownIssueContext, in eventContext: borrowing Event.Context) {
```

Comment:
Looks like the `eventContext` parameter might be unused?

Reply:
Correct, but I left the parameter here to match EncodedAttachment and EncodedError which also take this parameter but don't use it.

Looking around further, I see that there are various ABI.Encoded* types that *don't* take this parameter, so maybe this isn't as much of a convention as I thought. I can remove it here. Should I remove it from EncodedAttachment and EncodedError as well?

Reply:
Hmm, maybe @grynspan knows the reason behind having that parameter. These types aren't publicly exposed though so I'd lean towards removing this parameter and leaving the others unmodified since they're not relevant to this PR.

Reply:
It should not be necessary to create this type in order to support this change.

Reply:
The `eventContext` paramenter is inconsistently present because I was inconsistent in including it. These types don't currently need it and I don't recall having a change in the wings that _would_, but if I do I'd add it in a future PR. Doesn't need to be in any changes you make (but also this change doesn't appear necessary in the first place.)

Reply:
OK, I will remove this parameter if this type survives. I added the type at @stmontgomery's suggestion: https://github.com/swiftlang/swift-testing/pull/1014#discussion_r2008239304

Reply:
I removed this type.